include/onnxruntime/core/framework/sparse_tensor.h

```diff
@@ -1,10 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#if !defined(DISABLE_SPARSE_TENSORS)
```

Comment:
Not particularly related to the subject, but I have added it to reduce the number of PRs being sent.

This one looks like a bug in clang (we use 14.0.x) to me.
Without this patch I am getting the following error:

```
contrib/libs/onnx_runtime/include/onnxruntime/core/framework/sparse_tensor.h:24:12: error: redefinition of 'SparseFormat'
enum class SparseFormat : uint32_t {

contrib/libs/onnx_runtime/onnxruntime/core/framework/sparse_tensor.cc:7:10: note: '/home/thegeorg/arcadia/contrib/libs/onnx_runtime/include/onnxruntime/core/framework/sparse_tensor.h' included multiple times, additional include site here
#include "core/framework/sparse_tensor.h"

contrib/libs/onnx_runtime/include/onnxruntime/core/framework/op_kernel.h:23:10: note: '/home/thegeorg/arcadia/contrib/libs/onnx_runtime/include/onnxruntime/core/framework/sparse_tensor.h' included multiple times, additional include site here
#include "core/framework/sparse_tensor.h"
```

// and same for `class SparseTensor final {`